### DISCUSSION TITLE
"Upload Proof of Your Academic Status" Redirecting to Itself

### DISCUSSION THREAD

**Discussion by ttcriswell** [community/community#127733] [GitHub Education]
### Hi everyone,

Currently trying to apply for the Student Developer Pack. When I go to submit my proof of academic status the page just re-loads and my application does not appear to be accepted.

My photo is not triggering any errors when uploaded as far as I can tell. I have tested this across multiple browsers and mobile browsers.

Thanks for any help.

**Comment by davevad93** [comment]
Hi @ttcriswell ,

Take a look at these discussions #126848 , #126276 , several users encountered similar issues.

**Comment by Abbas-Razaa** [comment]
Troubleshooting Steps:
Check File Requirements:

Ensure that your proof of academic status meets all the file requirements (format, size, clarity).
Acceptable formats typically include JPEG, PNG, or PDF, and the file size should not exceed the specified limit.
Clear Cache and Cookies:

Sometimes, clearing your browser's cache and cookies can resolve issues with web pages not functioning correctly.
Try Different Browsers/Devices:

You mentioned you've tried multiple browsers and mobile browsers. Ensure that they are all updated to the latest versions.
Also, try using an incognito/private browsing mode to eliminate any potential interference from browser extensions.
Check Internet Connection:

Ensure that you have a stable internet connection while submitting your application.
Disable Ad-blockers and Extensions:

Sometimes, browser extensions, especially ad-blockers, can interfere with web pages. Try disabling them temporarily while submitting your application.
Upload a Different Image/File:

Try uploading a different proof of academic status to see if the issue is with the specific file you are trying to upload.
Contact Support:

If none of the above steps work, contact the support team for the Student Developer Pack. Provide them with detailed information about the issue, including the steps you have taken to try to resolve it.
Contacting Support
You can usually find a support or help section on the website where you are applying for the Student Developer Pack. Look for a contact form, email address, or live chat option to get in touch with their support team.

Additional Tips
Screenshot the Issue: When contacting support, include screenshots of the issue you are facing. This can help them understand and resolve the problem more efficiently.
Check Academic Institution’s Email: Ensure that you are using your academic institution’s email address for the application, as this is often required for verification.
By following these steps, you should be able to troubleshoot the issue and get assistance if needed. If you provide me with more specific details about the proof of academic status you are trying to upload, I may be able to offer more tailored advice.

**Reply by ttcriswell** [reply]
Thanks for your detailed response. 

I don't see the option to upload a pdf - though that would be ideal. I have a suspicion that the issue is somewhere in my file-type. The page I have been on seems to require a .jpeg/.png of a certain resolution under 1 mb. I think my attempts to scale down the size of my acceptance letter is triggering a resolution error (and the error is not visible on my end). 

Uploading the pdf directly would be ideal. Below is a screenshot of the page I am on.
<img width="1145" alt="Screenshot 2024-06-07 at 11 18 25 AM" src="https://github.com/community/community/assets/47951670/d2e53a75-519b-49cd-8c56-56d9321417d4">

**Reply by davevad93** [reply]
Hey @ttcriswell maybe you've already tried it but if you haven't, try taking a picture with your mobile (which is better than the PC webcam) instead of choosing the upload option, it may fix the issue.
